Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -64,7 +64,7 @@ void SkylanderPortalWindow::CreateMainWindow()
   checkbox_layout->setAlignment(Qt::AlignHCenter);
   m_checkbox = new QCheckBox(tr("Emulate Skylander Portal"), this);
   m_checkbox->setChecked(Config::Get(Config::MAIN_EMULATE_SKYLANDER_PORTAL));
-  connect(m_checkbox, &QCheckBox::toggled, [=](bool checked) { EmulatePortal(checked); });
+  connect(m_checkbox, &QCheckBox::toggled, [&](bool checked) { EmulatePortal(checked); });
```

Comment:
Is this right?

My understanding is that `&` captures `this` by reference, which is invalid as soon as this scope ends, and the lambda lives on. beyond the scope. 

Even if it is correct, [this] is clearer.

Reply:
Ok, did some research. 

Apparently since c++17, `[&]` automatically copies `*this` by reference (instead of `this` by reference as I was expecting)